includes/utilities.inc

```diff
@@ -364,11 +366,20 @@ function islandora_solr_prepare_solr_results($solr_results) {
           $tf =& $truncate_length[$field];
           $wordsafe = FALSE;
           $min_wordsafe_length = 1;
+
           if (isset($tf['wordsafe'])) {
             $wordsafe = $tf['wordsafe'];
             $min_wordsafe_length = $tf['wordsafe_length'];
           }
-          $val_val = truncate_utf8($val_val, $tf['maximum_length'], $wordsafe, $tf['add_ellipsis'], $min_wordsafe_length);
+
+          $val_val = new IslandoraSolrFieldTruncation(
```

Comment:
Should we perhaps be casting to string at this point? Didn't want to do the level of digging necessary to figure out if there are any cases in here or modules out there that might be expecting a string here.